template/.gitattributes

```diff
@@ -1,10 +1,10 @@
+/docs export-ignore
+/test export-ignore
```

Comment:
Why did you change this? The values were sorted via `natsort` previously...

Reply:
Alphabetical order, directories first, the same we have in .gitignore
Also the same order I can see in PHPStorm project view.

Reply:
That would be more clear if directories had a `/` suffix; without it, on unix systems at least, files do not need suffixes, which means it's hard to determine if the entry is a directory or a file.

Alphabetical order is the most important, as far as I'm concerned; it makes finding the entry in the list easier.